src/managers/PromptsManager.ts

```diff
@@ -310,20 +329,43 @@ export class PromptsManager {
   private getDelayedPromptOptions(promptOptions: AppUserConfigPromptOptions,
       type: DelayedPromptType
     ): DelayedPromptOptions {
-    const promptOptionsForSpecificType = promptOptions[type];
-    if (!promptOptions || !promptOptionsForSpecificType) {
-      return {
+      const defaultOptions = {
         enabled: false,
         autoPrompt: false,
         timeDelay: SERVER_CONFIG_DEFAULTS_PROMPT_DELAYS.timeDelay,
         pageViews: SERVER_CONFIG_DEFAULTS_PROMPT_DELAYS.pageViews
       };
-    }
-    return {
-      enabled: promptOptionsForSpecificType.enabled,
-      autoPrompt: promptOptionsForSpecificType.autoPrompt,
-      timeDelay: promptOptionsForSpecificType.timeDelay,
-      pageViews: promptOptionsForSpecificType.pageViews
-    };
+
+      if (!promptOptions || !promptOptions.native || !promptOptions.slidedown) {
+        // default
+        return defaultOptions;
+      }
+
+      switch (type) {
+        case DelayedPromptType.Native:
+          const nativePromptOptions = promptOptions.native;
+          return {
+            enabled: nativePromptOptions?.enabled,
+            autoPrompt: nativePromptOptions?.autoPrompt,
+            timeDelay: nativePromptOptions?.timeDelay,
+            pageViews: nativePromptOptions?.pageViews
+          };
+        case DelayedPromptType.Push:
+          case DelayedPromptType.Category:
+            case DelayedPromptType.Email:
+              case DelayedPromptType.Sms:
+                case DelayedPromptType.SmsAndEmail:
```

Comment:
This case and all above should have the same spacing so they all line up.
Or if there some new TypeScript way to check for more than one value in a single case statement